internal/exec/utils.go

```diff
@@ -576,9 +577,28 @@ func processArgsAndFlags(inputArgsAndFlags []string) (c.ArgsAndFlagsInfo, error)
 	}
 
 	if len(additionalArgsAndFlags) > 1 {
-		// Handle the legacy command `terraform write varfile`
-		if additionalArgsAndFlags[0] == "write" && additionalArgsAndFlags[1] == "varfile" {
-			info.SubCommand = "write varfile"
+		twoWordsCommand := false
+
+		// Handle terraform two-words commands
+		// https://www.terraform.io/cli/commands
+		if componentType == "terraform" {
+			// Handle the custom legacy command `terraform write varfile` (NOTE: use `terraform generate varfile` instead)
```

Comment:
Is `write varfile` legacy ?

Reply:
it was implemented before `atmos terraform generate ...` commands, so it's not recommended to use.

Use `atmos terraform generate varfile` to confirm to the generate/describe/validate patterns.